Sources/Mocker.swift

```diff
@@ -110,4 +116,20 @@ public struct Mocker {
             return shared.mocks.first(where: { $0 == request })
         }
     }
+
+    /// The way Mocker handles unregistered urls
+    public enum Mode {
```

Comment:
Can you move this enum to the top of the Mocker definition instead of here at the bottom?

Reply:
Ok, will do.